### ISSUE TITLE
@v1.1: Error: Cannot find module '@actions/core'

### ISSUE THREAD

**Issue by ychen4-li** [Azure/sql-action#70]
Hi there,

I am getting below error when running with @v1.1:

Error: Cannot find module '@actions/core'
```
Require stack:
- /actions-runner/_work/_actions/azure/sql-action/v1.1/lib/main.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/actions-runner/_work/_actions/azure/sql-action/v1.1/lib/main.js:34:27)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/actions-runner/_work/_actions/azure/sql-action/v1.1/lib/main.js'
  ]
}
```
It seems like the `node_modules` directory is missing from the v1.1 release? Can you help to take a look?

Thanks,
Yi

**Comment by dzsquared** [comment]
Which environment/runner type is this occurring on?

**Comment by ychen4-li** [comment]
> Which environment/runner type is this occurring on?

self hosted runner with Ubuntu 20.04 LTS

**Comment by fnmarquez** [comment]
Same error here but with GitHub-hosted runner (Ubuntu 20.04.3 LTS)

**Comment by dzsquared** [comment]
Fixed in v1.2, thank you for raising this issue!  I'll be keeping an eye out here if you continue to experience troubles.
https://github.com/Azure/sql-action/releases/tag/v1.2
